lib/commands/recordscreen.js

```diff
@@ -67,15 +68,31 @@ class ScreenRecorder {
       '-y', this.videoPath
     );
     this.mainProcess = new SubProcess(FFMPEG_BINARY, args);
+    let isCaptureStarted = false;
     this.mainProcess.on('output', (stdout, stderr) => {
-      if (stderr && !stderr.includes('frame=')) {
-        ffmpegLogger.info(`${stderr}`);
+      if (stderr) {
```

Comment:
Why not use this in a start detector on `start`? https://github.com/appium/node-teen_process#start-detectors

Reply:
because if startDetector fails then the whole process is considered invalid. Although, I assume the process might be still running, but it just didn't have enough time for the initialisation.